packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -183,6 +184,14 @@ export class DatePicker {
     return this.duetInput.hide(moveFocusToButton);
   }
 
+  /**
+   * Watch Value property for changes from outside and change hasValue based on that
```

Comment:
I would change the comment a bit, for consistency with the rest of the file, something like this:

```
Watch `value` property for changes and update `hasValue` based on that.
```

Reply:
Agreed makes it more readable too